### ISSUE TITLE
Cannot find issue for AG-10897 in change log. 

### ISSUE THREAD

**Issue by yuuk1993** [ag-grid/ag-grid#7935]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[x] support request => see 'Requesting Community Support'
```

Hello there,
I would like to know the details of AD-10897
([Regression] When toggling wrapText and autoHeight, cell values from different rows get rendered over one another)

Is it possible to share more details?
Especially I’d like to know bug occurred condition (Toggling autoHight means the the value of autoHight itself is changed? Or like triggered height change by autoHight?), in which version is the bug occurred version and how this affects. 

Currently I am facing an issue that the cell value is set to different row, which found it similar with this issue. 

Ag-grid version: 30.1.0
Framework: Vue

Thanks in advance!

**Comment by AG-Zoheil** [comment]
Hi @yuuk1993,

This was a regression introduced after `v30.2.1`. So `v30.2.1` was the last working version before the fix was applied in our latest release `v31.3.1`.

Here is an example from `v31.2` where you can see the effect of the bug: https://plnkr.co/edit/Rq3edSzXcKAdUFmZ?open=main.js

1. Wrap column 2 then scroll to the bottom of the grid
2. Wrap column 1
3. Clip Clip 1
**Result:** You can see values overlapping each other

I hope this helps.

Please note if you are experiencing a different issue from the one described above, we would need a plunker that reproduces the problem to investigate this in the most efficient manner.

Kind regards,
Zoheil

**Comment by yuuk1993** [comment]
Hi Zoheil!

Thank you for your early reply!
I found that it is not what I thought and my issue. 

I m still on an investigation for my issue so might ask you further question if any. 

Best wishes,
Yuriko.
